templates/CRM/Sepa/Page/CloseGroup.tpl

```diff
@@ -19,7 +19,7 @@
 			{capture assign=group_text}
 			<font size="+1">{ts domain="org.project60.sepa"}Group '%s' <b>is now closed</b> and cannot be changed any more.{/ts}</font>
 			{/capture}
-			<p>{$group_text|sprintf:$txgroup.reference}</p>
+			<p>{$group_text|string_format:$txgroup.reference}</p>
```

Comment:
Should be `$txgroup.reference|string_format:$group_text` because `$group_text` contains the format.

See https://www.smarty.net/docs/en/language.modifier.string.format.tpl